### PR TITLE
SDK 2.0 AssetDetailsPanel

### LINKED ISSUE #330: Move AssetDetailsPanel to new SDK
Move AssetDetailsPanel to new SDK

### PR DESCRIPTION
**Pull request by xitman78**
* HOC withAsset has been moved to SDK 2.0
* AssetDetailsPanel - SDK 2.0
* AssetMeta - support for new SDK (partially in test)

Closes #330

### CODE REVIEW

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-335.gearboxjs.preview.cogniteapp.com

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-335.gearboxjs.preview.cogniteapp.com

File: src/components/AssetMeta/AssetMeta.test.tsx [hunk 1]
```diff
@@ -28,9 +35,24 @@ beforeEach(() => {
   sdk.TimeSeries.list.mockResolvedValue({ items: timeseriesList });
 });
 
+afterEach(() => {
+  // @ts-ignore
+  fakeClient.assets.retrieve.mockClear();
```
[the remark this answers is not in the document]
Comment (xitman78) [reply src/components/AssetMeta/AssetMeta.test.tsx]: done

**Comment by f1cognite** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-335.gearboxjs.preview.cogniteapp.com

**Pull request merged**